frontend/package.json

```diff
@@ -18,16 +18,19 @@
   },
   "devDependencies": {
     "prettier": "1.18.2",
-    "react-scripts": "3.0.1"
+    "react-scripts": "3.1.1"
   },
   "resolutions": {
-    "babel-eslint": "10.0.1",
-    "braces": ">=2.3.1",
-    "handlebars": ">=4.0.13",
+    "babel-eslint": "10.0.2",
```

Comment:
I suspect we can drop this line, since ``react-scripts`` is so set on exactly this version.

Reply:
I'll try that out.